packages/azure/kibana/dashboard/azure-5ee36c30-32dc-11ed-a2e6-916b60bbea71.json

```
@@ -1,748 +1,809 @@
-                "data_stream.dataset": "azure.identityprotection"
-              }
-            }
-          }
-        ],
-        "query": {
-          "language": "kuery",
-          "query": ""
-        }
-      }
-    },
-    "optionsJSON": {
-      "hidePanelTitles": false,
-      "syncColors": false,
-      "useMargins": true
-    },
-    "panelsJSON": [
-      {
-        "version": "8.6.0",
-        "type": "lens",
-        "gridData": {
-          "h": 10,
-          "i": "0705b471-583f-4593-916e-46b213966691",
-          "w": 8,
-          "x": 0,
-          "y": 0
-        },
-        "panelIndex": "0705b471-583f-4593-916e-46b213966691",
-        "embeddableConfig": {
-          "attributes": {
-            "references": [
-              {
-                "id": "logs-*",
-                "name": "indexpattern-datasource-current-indexpattern",
-                "type": "index-pattern"
-              },
-              {
-                "id": "logs-*",
-                "name": "indexpattern-datasource-layer-2b0a18e6-25a6-40ef-ade8-5dddbd897856",
-                "type": "index-pattern"
-              }
-            ],
-            "state": {
-              "datasourceStates": {
-                "formBased": {
-                  "layers": {
-                    "2b0a18e6-25a6-40ef-ade8-5dddbd897856": {
-                      "columnOrder": [
-                        "a1c2991d-9586-4c81-893a-e29584070568",
-                        "bd75d029-b44e-4c22-b06e-df0e4da1e694"
-                      ],
-                      "columns": {
-                        "a1c2991d-9586-4c81-893a-e29584070568": {
-                          "customLabel": true,
-                          "dataType": "string",
-                          "isBucketed": true,
-                          "label": "Source",
-                          "operationType": "terms",
-                          "params": {
-                            "missingBucket": false,
-                            "orderBy": {
-                              "columnId": "bd75d029-b44e-4c22-b06e-df0e4da1e694",
-                              "type": "column"
+    "attributes": {
+        "description": "Provide an overview and statistics of the provisioning activities on your enterprise applications.",
+        "kibanaSavedObjectMeta": {
+            "searchSourceJSON": {
+                "filter": [
+                    {
+                        "$state": {
+                            "store": "appState"
+                        },
+                        "meta": {
+                            "alias": null,
+                            "disabled": false,
+                            "indexRefName": "kibanaSavedObjectMeta.searchSourceJSON.filter[0].meta.index",
+                            "key": "data_stream.dataset",
+                            "negate": false,
+                            "params": {
+                                "query": "azure.identityprotection"
```

Update dataset to fix the dashboard:

```suggestion
                                "query": "azure.identity_protection"
```